includes/Modules/Search_Console.php

```diff
@@ -246,6 +246,20 @@ protected function create_data_request( Data_Request $data ) {
 		return new WP_Error( 'invalid_datapoint', __( 'Invalid datapoint.', 'google-site-kit' ) );
 	}
 
+	/**
+	 * Returns a punycode version of a unicode URL.
+	 *
+	 * @param string $url The URL to decode.
```

Comment:
This needs a `@since n.e.x.t` and an empty line above the `param` annotation.

Reply:
👍

Reply:
Assuming `@since n.e.x.t` gets replaced during the release process?